src/server.js

```diff
@@ -321,11 +321,15 @@ class Server {
     })
     .then(() => {
       return Promise.map(events.keySeq(), (exchange) => {
+        const worker = events.get(exchange)
+        const options = this._workerOptions[exchange] || {}
+        const queueOptions = options.queueOptions
         return this._rabbitmq.subscribeToFanoutExchange(
           exchange,
           (job, jobMeta, done) => {
-            this._enqueue(exchange, events.get(exchange), job, jobMeta, done)
-          }
+            this._enqueue(exchange, worker, job, jobMeta, done)
+          },
+          queueOptions
```

Comment:
No, that function already supported all needed options. Basically that functionality was already exposed for the publishers when they create exchanges and queues. But it was hidden for consumers that create queues on the subscribe for exchanges.